build-system/tasks/helpers.js

```diff
@@ -388,17 +430,29 @@ async function esbuildCompile(srcDir, srcFilename, destDir, options) {
       name: 'remap-dependencies',
       setup(build) {
         build.onResolve({filter: /.*/}, (args) => {
-          const {path: importPath, resolveDir} = args;
+          const {resolveDir} = args;
+          let {path: importPath} = args;
+          /**
```

Comment:
nit: `/** ... */` is generally used to indicate JSDoc comments annotating variables, functions, classes, files, etc. Since this is more of a "regular" code comment, prefer `//`

Reply:
changed to `//`